src/app/components/Caption/__snapshots__/index.test.tsx.snap

```diff
@@ -395,12 +415,11 @@ exports[`Captions should render correctly with inline block 1`] = `
           This is some text.
         </b>
         <span
-          class="emotion-3 emotion-4"
-          lang="fa"
```

Comment:
Just noticing this is removed. Do we need to retain the `lang` on this element?

Reply:
Soz, was an oversight on my part, have aded it back in.